### PR TITLE
Simulation legend

### LINKED ISSUE #291: Add legend to simulate page to clarify the colour encodings used by the panels
white is current run
blue is previous runs
orange is aggregate

### PR DESCRIPTION
**Pull request by YohannParis**
**What**
- fix #291 

**How**
- Separated the new model legends, into a legend panel to contains legends and a model legend.
- Created a new simulation legend for the runs.

**Videos**

https://user-images.githubusercontent.com/636801/128936065-5ea672b1-a93f-403e-adcb-cb4ee433cd31.mov

### CODE REVIEW

File: client/src/views/Simulation/components/SimulationLegend.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<template>
+  <ul class="legend-simulation">
+    <li class="run current">Current Run</li>
+    <li class="run previous">Previous Runs</li>
+    <li class="run aggregate">Aggregate of all Runs</li>
```
Comment (RosaRomeroGomez) [inline client/src/views/Simulation/components/SimulationLegend.vue:5]: We probably want to add here the line for observational data (even if we don't have it yet). I think the color should be `#E75BCD` but @emilykuang can confirm.

**Review by RosaRomeroGomez — APPROVED** [review]
Good idea to just have everything in the same legend. Looks great.

File: client/src/views/Simulation/components/SimulationLegend.vue (hunk 1 above)
Comment (YohannParis) [reply client/src/views/Simulation/components/SimulationLegend.vue:5]: Thanks, I was thimking that 3 legend buttons would be too much.

Do you think we need subtitles for the legend? I'm worriewd that user might think the runs colours are for edges. @emilykuang what are your thoughts?

Comment (emilykuang) [reply client/src/views/Simulation/components/SimulationLegend.vue:5]: Thanks for adding the observational data. Yes, the subtitles are good for differentiating between the Model Panel and Variable Panel, the divider looks great! 
I am just wondering about the wording for the yellow line: the meaning of `Aggregate` is "calculated by the combination of units; total" which may be confusing, many a better word is `Average`?

**Review by emilykuang — APPROVED** [review]

File: client/src/views/Simulation/components/SimulationLegend.vue (hunk 1 above)
Comment (RosaRomeroGomez) [reply client/src/views/Simulation/components/SimulationLegend.vue:5]: 100%

Comment (YohannParis) [reply client/src/views/Simulation/components/SimulationLegend.vue:5]: Screen Shot 2021-08-11 at 09 30 05

**Pull request merged**